tests/TestHarness/launcher.py

```diff
@@ -521,9 +523,34 @@ def construct_command_line(self, instance: nodeDefinition):
             i = self.args.specific_nums.index(instance.index)
             specifics = getattr(self.args, f'specific_{Utils.EosServerName}es')[i]
             if specifics[0] == "'" and specifics[-1] == "'":
-                eosdcmd.extend(shlex.split(specifics[1:-1]))
+                specificList = shlex.split(specifics[1:-1])
             else:
-                eosdcmd.extend(shlex.split(specifics))
+                specificList = shlex.split(specifics)
+            # Allow specific nodeos args to override existing args up to this point.
+            # Consider moving specific arg handling to the end to allow overriding all args.
+            repeatable = [
+                # appbase
+                '--plugin',
+                # chain_plugin
+                '--checkpoint', '--profile-account', '--actor-whitelist', '--actor-blacklist',
+                '--contract-whitelist', '--contract-blacklist', '--action-blacklist', '--key-blacklist',
+                '--sender-bypass-whiteblacklist', '--trusted-producer',
+                # http_plugin
+                '--http-alias',
+                # net_plugin
+                '--p2p-peer-address', '--p2p-auto-bp-peer', '--peer-key', '--peer-private-key',
+                # producer_plugin
+                '--producer-name', '--signature-provider', '--greylist-account', '--disable-subjective-account-billing',
+                # trace_api_plugin
+                '--trace-rpc-abi']
```

Comment:
This seems a little kludgy and maintenance/error prone. But I guess it is okay.

Reply:
It's not perfect but I thought parsing the C++ for all the plugins was asking a bit much.  The options don't change too often and when they do the vast majority are not composeable so it shouldn't be too much of a pain.

If boost wasn't so wretched about allowing option overrides this code wouldn't exist...